pose_est_nets/utils/plotting_utils.py

```diff
@@ -39,7 +96,9 @@ def saveNumericalPredictions(model, datamod, threshold):
     full_dl = datamod.full_dataloader()
     test_dl = datamod.test_dataloader()
     final_gt_keypoints = np.empty(shape=(len(test_dl), model.num_keypoints, 2))
-    final_imgs = np.empty(shape=(len(test_dl), 406, 396, 1))
+    final_imgs = np.empty(
+        shape=(len(test_dl), 406, 396, 1)
```

Comment:
can we pull these numbers from the model somehow instead of hard-coding them?

Reply:
this is an unused and outdated function, i'd either remove it or ask @nickgreenspan to fix it?
I'd save stuff as a `.csv` that looks exactly like the labels csv.